internal/policy/policy.go

```diff
@@ -570,21 +566,57 @@ func (s *State) GetRootKeys() ([]tuf.Principal, error) {
 }
 
 // GetRootMetadata returns the deserialized payload of the State's RootEnvelope.
-func (s *State) GetRootMetadata() (tuf.RootMetadata, error) {
+// The `migrate` parameter determines if the schema must be converted to a newer
+// version.
```

Comment:
it shouldn't matter, this is purely an internal control. It's only there because eventually, we should only spend resources migrating when we're in a flow where policy metadata is being updated (adding a rule etc.). We should not migrate when simply loading something for verification.

Reply:
In fact, the exposed API surface should give no control over the version of metadata that is created.